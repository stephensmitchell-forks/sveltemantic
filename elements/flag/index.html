<!doctype html> <head> <meta charset=utf-8> <meta content="width=device-width,initial-scale=1" name=viewport> <meta content=#333333 name=theme-color> <base href=/sveltemantic/ > <link href=manifest.json rel=manifest> <link href=favicon.png rel=icon type=image/png> <link href=client/main.499961639.css rel=stylesheet> <link href=global.css rel=stylesheet> <noscript id=sapper-head-start></noscript><title>404</title><noscript id=sapper-head-end></noscript> </head> <body class=pushable id=example> <div class=pusher> <div class="full height"> <div class=toc> <div class="ui vertical inverted menu"> <div class=item> <a href=./ class="ui icon image logo"> <img alt="SvelteMantic Logo" src=logo.png> </a> <a href=/ ><b>UI Docs</b></a> </div> <a href=getting-started class=item rel=prefetch> <b>Getting Started</b> </a> <div class=item> <div class=header>Elements</div> <div class=menu> <a href=elements/button class=item rel=prefetch> Button </a><a href=elements/container class=item rel=prefetch> Container </a><a href=elements/divider class=item rel=prefetch> Divider </a><a href=elements/flag class="item active" rel=prefetch> Flag </a><a href=elements/group class=item rel=prefetch> Group </a><a href=elements/header class=item rel=prefetch> Header </a><a href=elements/icon class=item rel=prefetch> Icon </a><a href=elements/image class=item rel=prefetch> Image </a><a href=elements/input class=item rel=prefetch> Input </a><a href=elements/label class=item rel=prefetch> Label </a><a href=elements/list class=item rel=prefetch> List </a><a href=elements/placeholder class=item rel=prefetch> Placeholder </a><a href=elements/rail class=item rel=prefetch> Rail </a><a href=elements/reveal class=item rel=prefetch> Reveal </a><a href=elements/segment class=item rel=prefetch> Segment </a><a href=elements/step class=item rel=prefetch> Step </a><a href=elements/text class=item rel=prefetch> Text </a><a href=elements/wrapper class=item rel=prefetch> Wrapper </a> </div> </div><div class=item> <div class=header>Collections</div> <div class=menu> <a href="collections/coming soon" class=item rel=prefetch> Coming Soon </a> </div> </div><div class=item> <div class=header>Views</div> <div class=menu> <a href="views/coming soon" class=item rel=prefetch> Coming Soon </a> </div> </div><div class=item> <div class=header>Modules</div> <div class=menu> <a href="modules/coming soon" class=item rel=prefetch> Coming Soon </a> </div> </div><div class=item> <div class=header>Behavior (Coming Soon)</div> <div class=menu> <a href="behavior (coming soon)/api" class=item rel=prefetch> API </a><a href="behavior (coming soon)/form validation" class=item rel=prefetch> Form Validation </a><a href="behavior (coming soon)/visibility" class=item rel=prefetch> Visibility </a> </div> </div> </div> </div> <div class=article> <div class="ui vertical masthead segment" vertical vertical=true> <div class="ui container"> <div class=introduction> <h1 class="ui header"> 404 <div class="header sub"> Not found </div> </h1> <div class="ui divider hidden"></div> </div> </div> </div> <div class="ui divider hidden"></div> <div class="ui container"> <div class="ui fluid placeholder"> <div class=header> <div class=line></div><div class=line></div><div class=line></div><div class=line></div> </div> <div class=paragraph> <div class=line></div><div class=line></div><div class=line></div><div class=line></div><div class=line></div> </div><div class=paragraph> <div class=line></div><div class=line></div><div class=line></div><div class=line></div><div class=line></div> </div><div class=paragraph> <div class=line></div><div class=line></div><div class=line></div><div class=line></div><div class=line></div> </div><div class=paragraph> <div class=line></div><div class=line></div><div class=line></div><div class=line></div><div class=line></div> </div><div class=paragraph> <div class=line></div><div class=line></div><div class=line></div><div class=line></div><div class=line></div> </div> </div> </div> </div> </div> </div> <script>__SAPPER__={error:{message:"Not found"},status:404,baseUrl:"/sveltemantic",preloaded:[void 0,{}]};if('serviceWorker' in navigator)navigator.serviceWorker.register('/sveltemantic/service-worker.js');(function(){try{eval("async function x(){}");var main="/sveltemantic/client/client.b124ca7a.js"}catch(e){main="/sveltemantic/client/legacy/client.00e9125a.js"};var s=document.createElement("script");try{new Function("if(0)import('')")();s.src=main;s.type="module";s.crossOrigin="use-credentials";}catch(e){s.src="/sveltemantic/client/shimport@1.0.1.js";s.setAttribute("data-main",main);}document.head.appendChild(s);}());</script> 