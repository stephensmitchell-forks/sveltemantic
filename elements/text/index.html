<!doctype html> <head> <meta charset=utf-8> <meta content="width=device-width,initial-scale=1" name=viewport> <meta content=#333333 name=theme-color> <base href=/sveltemantic/ > <link href=manifest.json rel=manifest> <link href=favicon.png rel=icon type=image/png> <link href=client/main.499961639.css rel=stylesheet><link href=client/text.df1cc6ea.css rel=stylesheet> <link href=global.css rel=stylesheet> <noscript id=sapper-head-start></noscript><title>Text | Sveltemantic</title><noscript id=sapper-head-end></noscript> </head> <body class=pushable id=example> <div class=pusher> <div class="full height"> <div class=toc> <div class="ui vertical inverted menu"> <div class=item> <a href=./ class="ui icon image logo"> <img alt="SvelteMantic Logo" src=logo.png> </a> <a href=/ ><b>UI Docs</b></a> </div> <a href=getting-started class=item rel=prefetch> <b>Getting Started</b> </a> <div class=item> <div class=header>Elements</div> <div class=menu> <a href=elements/button class=item rel=prefetch> Button </a><a href=elements/container class=item rel=prefetch> Container </a><a href=elements/divider class=item rel=prefetch> Divider </a><a href=elements/flag class=item rel=prefetch> Flag </a><a href=elements/group class=item rel=prefetch> Group </a><a href=elements/header class=item rel=prefetch> Header </a><a href=elements/icon class=item rel=prefetch> Icon </a><a href=elements/image class=item rel=prefetch> Image </a><a href=elements/input class=item rel=prefetch> Input </a><a href=elements/label class=item rel=prefetch> Label </a><a href=elements/list class=item rel=prefetch> List </a><a href=elements/placeholder class=item rel=prefetch> Placeholder </a><a href=elements/rail class=item rel=prefetch> Rail </a><a href=elements/reveal class=item rel=prefetch> Reveal </a><a href=elements/segment class=item rel=prefetch> Segment </a><a href=elements/step class=item rel=prefetch> Step </a><a href=elements/text class="item active" rel=prefetch> Text </a><a href=elements/wrapper class=item rel=prefetch> Wrapper </a> </div> </div><div class=item> <div class=header>Collections</div> <div class=menu> <a href="collections/coming soon" class=item rel=prefetch> Coming Soon </a> </div> </div><div class=item> <div class=header>Views</div> <div class=menu> <a href="views/coming soon" class=item rel=prefetch> Coming Soon </a> </div> </div><div class=item> <div class=header>Modules</div> <div class=menu> <a href="modules/coming soon" class=item rel=prefetch> Coming Soon </a> </div> </div><div class=item> <div class=header>Behavior (Coming Soon)</div> <div class=menu> <a href="behavior (coming soon)/api" class=item rel=prefetch> API </a><a href="behavior (coming soon)/form validation" class=item rel=prefetch> Form Validation </a><a href="behavior (coming soon)/visibility" class=item rel=prefetch> Visibility </a> </div> </div> </div> </div> <div class=article> <div class="ui segment masthead vertical" vertical vertical=true> <div class="ui container"> <div class=introduction> <h1 class="ui header"> Text <div class="header sub"> A text is used to style some inline text with a simple color </div> </h1> <div class="ui divider hidden"></div> </div> </div> </div> <div class="ui divider hidden"></div> <div class="ui container main"> <div class="ui active tab" data-tab=text> <h2 class="ui header"> Definition </h2> <div class=example> <h4 class="ui header" id=size> <a href=#size> <i class="icon fitted linkify small"></i> Size </a> </h4> <i class="icon fitted code"></i> <p>Text can vary in the same sizes as icons</p> <div class="svelte-sf8mkp html"> <segment> <p>Starting with <text color=red size=mini>mini</text> text</p> <p>which turns into <span class="ui red text tiny">tiny</span> text</p> <p>changing to <span class="ui red text small">small</span> text until it is</p> <p>the default <span class="ui red text medium">medium</span> text</p> <p>and could be <span class="ui red text large">large</span> text</p> <p>to turn into <span class="ui red text big">big</span> text</p> <p>then growing to <span class="ui red text huge">huge</span> text</p> <p>to finally become <span class="ui red text massive">massive</span> text</p> </segment> <div class="ui attached label top">Example <i class="icon copy link"></i></div> </div> <div class="hidden annotation svelte-sf8mkp transition"> <div class="ui attached bottom instructive segment"> <pre>
	                
            <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;</span>Segment</span><span class="token punctuation">></span></span>
                <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;</span>p</span><span class="token punctuation">></span></span>Starting with <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;</span>Text</span> <span class="token attr-name">size</span><span class="token attr-value"><span class="token punctuation">=</span><span class="token punctuation">"</span>mini<span class="token punctuation">"</span></span> <span class="token attr-name">color</span><span class="token attr-value"><span class="token punctuation">=</span><span class="token punctuation">"</span>red<span class="token punctuation">"</span></span><span class="token punctuation">></span></span>mini<span class="token tag"><span class="token tag"><span class="token punctuation">&lt;/</span>Text</span><span class="token punctuation">></span></span> text<span class="token tag"><span class="token tag"><span class="token punctuation">&lt;/</span>p</span><span class="token punctuation">></span></span>
                <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;</span>p</span><span class="token punctuation">></span></span>which turns into <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;</span>span</span> <span class="token attr-name">class</span><span class="token attr-value"><span class="token punctuation">=</span><span class="token punctuation">"</span>ui tiny red text<span class="token punctuation">"</span></span><span class="token punctuation">></span></span>tiny<span class="token tag"><span class="token tag"><span class="token punctuation">&lt;/</span>span</span><span class="token punctuation">></span></span> text<span class="token tag"><span class="token tag"><span class="token punctuation">&lt;/</span>p</span><span class="token punctuation">></span></span>
                <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;</span>p</span><span class="token punctuation">></span></span>changing to <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;</span>span</span> <span class="token attr-name">class</span><span class="token attr-value"><span class="token punctuation">=</span><span class="token punctuation">"</span>ui small red text<span class="token punctuation">"</span></span><span class="token punctuation">></span></span>small<span class="token tag"><span class="token tag"><span class="token punctuation">&lt;/</span>span</span><span class="token punctuation">></span></span> text until it is<span class="token tag"><span class="token tag"><span class="token punctuation">&lt;/</span>p</span><span class="token punctuation">></span></span>
                <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;</span>p</span><span class="token punctuation">></span></span>the default <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;</span>span</span> <span class="token attr-name">class</span><span class="token attr-value"><span class="token punctuation">=</span><span class="token punctuation">"</span>ui medium red text<span class="token punctuation">"</span></span><span class="token punctuation">></span></span>medium<span class="token tag"><span class="token tag"><span class="token punctuation">&lt;/</span>span</span><span class="token punctuation">></span></span> text<span class="token tag"><span class="token tag"><span class="token punctuation">&lt;/</span>p</span><span class="token punctuation">></span></span>
                <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;</span>p</span><span class="token punctuation">></span></span>and could be <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;</span>span</span> <span class="token attr-name">class</span><span class="token attr-value"><span class="token punctuation">=</span><span class="token punctuation">"</span>ui large red text<span class="token punctuation">"</span></span><span class="token punctuation">></span></span>large<span class="token tag"><span class="token tag"><span class="token punctuation">&lt;/</span>span</span><span class="token punctuation">></span></span> text<span class="token tag"><span class="token tag"><span class="token punctuation">&lt;/</span>p</span><span class="token punctuation">></span></span>
                <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;</span>p</span><span class="token punctuation">></span></span>to turn into <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;</span>span</span> <span class="token attr-name">class</span><span class="token attr-value"><span class="token punctuation">=</span><span class="token punctuation">"</span>ui big red text<span class="token punctuation">"</span></span><span class="token punctuation">></span></span>big<span class="token tag"><span class="token tag"><span class="token punctuation">&lt;/</span>span</span><span class="token punctuation">></span></span> text<span class="token tag"><span class="token tag"><span class="token punctuation">&lt;/</span>p</span><span class="token punctuation">></span></span>
                <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;</span>p</span><span class="token punctuation">></span></span>then growing to <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;</span>span</span> <span class="token attr-name">class</span><span class="token attr-value"><span class="token punctuation">=</span><span class="token punctuation">"</span>ui huge red text<span class="token punctuation">"</span></span><span class="token punctuation">></span></span>huge<span class="token tag"><span class="token tag"><span class="token punctuation">&lt;/</span>span</span><span class="token punctuation">></span></span> text<span class="token tag"><span class="token tag"><span class="token punctuation">&lt;/</span>p</span><span class="token punctuation">></span></span>
                <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;</span>p</span><span class="token punctuation">></span></span>to finally become <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;</span>span</span> <span class="token attr-name">class</span><span class="token attr-value"><span class="token punctuation">=</span><span class="token punctuation">"</span>ui red massive text<span class="token punctuation">"</span></span><span class="token punctuation">></span></span>massive<span class="token tag"><span class="token tag"><span class="token punctuation">&lt;/</span>span</span><span class="token punctuation">></span></span> text<span class="token tag"><span class="token tag"><span class="token punctuation">&lt;/</span>p</span><span class="token punctuation">></span></span>
            <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;/</span>Segment</span><span class="token punctuation">></span></span>
        
	            </pre> </div> </div> </div> </div> </div> </div> </div> </div> <script>__SAPPER__={baseUrl:"/sveltemantic",preloaded:[void 0,null,{}]};if('serviceWorker' in navigator)navigator.serviceWorker.register('/sveltemantic/service-worker.js');(function(){try{eval("async function x(){}");var main="/sveltemantic/client/client.b124ca7a.js"}catch(e){main="/sveltemantic/client/legacy/client.00e9125a.js"};var s=document.createElement("script");try{new Function("if(0)import('')")();s.src=main;s.type="module";s.crossOrigin="use-credentials";}catch(e){s.src="/sveltemantic/client/shimport@1.0.1.js";s.setAttribute("data-main",main);}document.head.appendChild(s);}());</script> 