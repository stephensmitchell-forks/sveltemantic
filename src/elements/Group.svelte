{#if icons}
    <i class="{size} icons"></i>
{:else if segments}
    <div class="ui segments" class:basic class:stacked class:horizontal class:piled class:raised
    class:compact>
        <slot></slot>
    </div>
{:else}
    <div class="ui {size} {direction} {attached} {color}" class:images class:icon class:buttons 
    class:labels={!buttons} class:labeled={buttons && labels} class:basic class:attached
    class:tag class:circular>
        <slot></slot>
    </div>
{/if}

<script>
    export let size = '';
    export let direction = '';
    export let color = '';
    export let attached = '';
    export let stacked = false;
    export let horizontal = false;
    export let segments = false;
    export let basic = false;
    export let compact = false;
    export let images = false;
    export let icon = false;
    export let tag = false;
    export let piled = false;
    export let raised = false;
    export let circular = false;
    export let buttons = false;
    export let labels = false;
    export let icons = false;
</script>
