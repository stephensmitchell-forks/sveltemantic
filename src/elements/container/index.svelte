<div
    use:actions={use}
    use:events
    class="{clsxd(
        'ui',
        [alignment, { 'aligned': alignment && alignment !== 'justified' }],
        fluid && 'fluid',
        text && 'text',
        className,
        'container'
    )}"
    {...exclude($$props, ['use', 'class', 'alignment', 'fluid', 'text'])}
><slot>No content is provided</slot></div>

<script context="module">
    import { exclude, eventsBuilder, actions, clsxd } from '../../lib'
    import { current_component } from 'svelte/internal'
</script>

<script>
    const events = eventsBuilder(current_component)

    export let use = [];
    let className = '';
    export { className as class };

    export let fluid = false;
    export let text = false;
    export let alignment = '';
</script>
