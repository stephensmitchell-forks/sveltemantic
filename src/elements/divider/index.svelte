<div
    use:actions={use}
    use:events
    class="{clsxd(
        'ui',
        direction,
        inverted && 'inverted',
        fitted && 'fitted',
        hidden && 'hidden',
        section && 'section',
        clearing && 'clearing',
        className,
        'divider'
    )}"
    {...exclude($$props, ['use', 'class', 'direction', 'inverted', 'fitted', 'hidden', 'section', 'clearing'])}
><slot></slot></div>

<script context="module">
    import { exclude, eventsBuilder, actions, clsxd } from '../../lib'
    import { current_component } from 'svelte/internal'
</script>

<script>
    const events = eventsBuilder(current_component)

    export let use = [];
    let className = '';
    export { className as class };

    export let direction = '';
    export let inverted = false;
    export let fitted = false;
    export let hidden = false;
    export let section = false;
    export let clearing = false;
</script>
