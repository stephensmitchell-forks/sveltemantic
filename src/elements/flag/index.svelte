<i
    use:actions={use}
    use:events
    class="{clsxd(
        code,
        name,
        alias,
        className,
        'flag'
    )}"
    {...exclude($$props, ['use', 'class', 'code', 'name', 'alias'])}
></i>

<script context="module">
    import { exclude, eventsBuilder, actions, clsxd } from '../../lib'
    import { current_component } from 'svelte/internal'
</script>

<script>
    const events = eventsBuilder(current_component)

    export let use = [];
    let className = '';
    export { className as class };

    export let code = '';
    export let name = '';
    export let alias = '';
</script>
