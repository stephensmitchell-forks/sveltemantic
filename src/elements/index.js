import * as Button from './button/index.svelte'
export { Button }

import * as Container from './container/index.svelte'
export { Container }

import * as Divider from './divider/index.svelte'
export { Divider }

import * as Flag from './flag/index.svelte'
export { Flag }

import * as Header from './header/index.svelte'
export { Header }

import * as Icon from './icon/index.svelte'
export { Icon }

import * as Image from './image/index.svelte'
export { Image }

import * as Input from './input/index.svelte'
export { Input }

import * as Loader from './loader/index.svelte'
export { Loader }

import * as Placeholder from './placeholder/index.svelte'
export { Placeholder }

import * as Rail from './rail/index.svelte'
export { Rail }

import * as Reveal from './reveal/index.svelte'
export { Reveal }

import * as Segment from './segment/index.svelte'
export { Segment }

import * as Step from './step/index.svelte'
export { Step }

import * as Text from './text/index.svelte'
export { Text }
