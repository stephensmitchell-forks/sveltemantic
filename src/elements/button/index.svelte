{#if animation}
    <div class="ui animated {animation} {color} {size} {floated} {attached} button" tabindex="0"  
    class:active class:tertiary class:basic class:inverted class:compact class:toggle class:fluid
    class:positive class:negative class:circular class:icon class:attached use:activate={toggle}>
        <div class="visible content">
            <slot name="visible">No visible content provided</slot>
        </div>
        <div class="hidden content">
            <slot name="hidden">No hidden content provided</slot>
        </div>
    </div>
{:else if labeled}
    <div class="ui {color} {size} {floated} {labeled} {attached} button" tabindex="0"  
    class:active class:tertiary class:basic class:inverted class:compact class:toggle class:fluid
    class:positive class:negative class:circular class:icon class:attached use:activate={toggle}>
        <slot></slot>
    </div>
{:else}
    <button class="ui {emphasis} {color} {loading} {size} {floated} {attached} button" 
    class:active class:tertiary class:basic class:inverted class:loading class:compact class:toggle class:fluid
    class:positive class:negative class:circular class:icon class:attached use:activate={toggle}>
        {#if loading}
            Loading
        {:else}
            <slot>
                {#if icon}
                    <Icon name={icon} />
                {/if}
            </slot>
        {/if}
    </button>
{/if}

<script>
    import { listen } from 'svelte/internal';
    import { smfloated, smlabeled } from '../../lib/elements/util';

    import Icon from '../icon/index.svelte';

    export let active = false;
    export let emphasis = '';
    export let color = '';
    export let animation = '';
    export let loading = '';
    export let size = '';
    export let inverted = false;
    export let basic = false;
    export let tertiary = false;
    export let compact = false;
    export let toggle = '';
    export let fluid = false;
    export let float = '';
    export let attached = '';
    export let label = '';
    export let positive = false;
    export let negative = false;
    export let circular = false;
    export let icon = '';

    $: floated = smfloated(float)
    $: labeled = smlabeled(label)

    function toggleState(e) {
        if(toggle) {
            active = !active;
        }
    }

    function activate(node, ev) {
        let _unlistener = typeof ev === 'string' && ev.startsWith('on:') ? listen(node, ev.substring(3), toggleState) : null;
        
        return {
            update(ev) {
                _unlistener = _unlistener ? _unlistener() : null
                _unlistener = typeof ev === 'string' && ev.startsWith('on:') ? listen(node, ev.substring(3), toggleState) : null;
            },

            destroy() {
                if(_unlistener) _unlistener();
            }
        }
    }
</script>
