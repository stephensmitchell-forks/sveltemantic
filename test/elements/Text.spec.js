import { render, cleanup } from '@testing-library/svelte'
import { Text } from '../../src/elements'

beforeEach(cleanup)

describe('Text.svelte', () => {
    test('dummy test', () => {
        expect(true).toEqual(true)
    })
})
