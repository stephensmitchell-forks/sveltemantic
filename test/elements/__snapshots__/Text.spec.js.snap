// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Text.svelte Text with additional attributes 1`] = `"<span class=\\"ui text\\" data-testid=\\"any\\"></span>"`;

exports[`Text.svelte Text with additional classes 1`] = `"<span class=\\"ui text-lato text-compact text\\" data-testid=\\"any\\"></span>"`;

exports[`Text.svelte Text with color prop 1`] = `"<span class=\\"ui red text\\" data-testid=\\"any\\"></span>"`;

exports[`Text.svelte Text with custom actions 1`] = `"<span class=\\"ui text drag-drop-enable\\" data-testid=\\"any\\" draggable=\\"true\\"></span>"`;

exports[`Text.svelte Text with default props 1`] = `"<span class=\\"ui text\\" data-testid=\\"any\\"></span>"`;

exports[`Text.svelte Text with disabled prop 1`] = `"<span class=\\"ui disabled text\\" data-testid=\\"any\\"></span>"`;

exports[`Text.svelte Text with dom events 1`] = `"<span class=\\"ui text\\" data-testid=\\"any\\" clicked=\\"true\\" focused=\\"true\\"></span>"`;

exports[`Text.svelte Text with inverted prop 1`] = `"<span class=\\"ui inverted text\\" data-testid=\\"any\\"></span>"`;

exports[`Text.svelte Text with multiple props and events snapshot 1`] = `"<span class=\\"ui purple inverted disabled extra text drag-drop-enable\\" data-testid=\\"any\\" draggable=\\"true\\" clicked=\\"true\\" focused=\\"true\\"></span>"`;

exports[`Text.svelte Text with size prop 1`] = `"<span class=\\"ui huge text\\" data-testid=\\"any\\"></span>"`;
